index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <nav id="navbar">
+    <header class="header" id="Documentation">Documentation</header>
+    <ul>
+      <li><a href="#introduction" class="nav-link">Introduction</a></li>
+      <li><a href="#what_should_we_already_know" class="nav-link">what should we already know</a></li>
+      <li><a href="#javascript_and_java" class="nav-link">Javascript and java</a></li>
+      <li><a href="#hello_world" class="nav-link">Hello world</a></li>
+      <li><a href="#variables" class="nav-link">variables</a></li>
+      <li><a href="#declaring_variables" class="nav-link">Declaring variables</a></li>
+    </ul>
+  </nav>
+
+  <main id="main-doc">
+    <section id="introduction" class="main-section">
+      <header>
+        Introduction
+      </header>
+
+      <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+      <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+      <ul>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+      </ul>
+
+    </section>
+
+    <section id="what_should_we_already_know" class="main-section">
+      <header>
+        What should we already know
+
+      </header>
+      <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+      <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+      <ul>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+        <li>Lorem ipsum</li>
+      </ul>
```

Comment:
You did a nice job outlining the `ul` items correctly but I think it could have been better if you put this `ul` in a `nav` container and place it in the header section because the `ul` items are navigation links and the `nav` element is intended only for major block of navigation links. Cheers!!! Good luck in your coding 👍